packages/asset-swapper/CHANGELOG.json

```
@@ -6,8 +6,7 @@
                 "note": "Changed Sushiswap router address",
                 "pr": 373
             }
-        ],
-        "timestamp": 1637349338
+        ]
```

```suggestion
        ],
        "timestamp": 1637349338
```

This log is append only, can't modify the previous entries.